src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -209,6 +212,8 @@ folder InstallDir:\MSBuild\Current\Bin\amd64
   file source=$(X86BinPath)Microsoft.Build.Tasks.Core.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)Microsoft.Build.Utilities.Core.dll vs.file.ngenArchitecture=all
   file source=$(X86BinPath)System.Buffers.dll vs.file.ngenArchitecture=all
+  file source=$(X86BinPath)System.Formats.Nrbf.dll vs.file.ngenArchitecture=all
```

Comment:
I'm seeing some new packages. Do we use these anywhere? I did not find any references with a quick search.

Reply:
Yeah, something (that I apparently didn't write down 🫠) added a dependency on these in the update. So our direct deps didn't change but the required closure did.